pom.client.xml

```diff
@@ -369,6 +370,10 @@
                 <title>Azure Key Vault</title>
                 <packages>com.azure.keyvault*</packages>
               </group>
+              <group>
+                <title>Azure Identity</title>
+                <packages>com.azure.identity*</packages>
+              </group>
```

Comment:
Sort these alphabetically - Identity before Key Vault

Reply:
Should they be sorted based on title or package? If it's based on package, event hub will move to the bottom and if it's the title, app config will move to the top. This decides the order in which the javadocs are actually generated. So, we can say, Core will be at the top and then all client library titles are alphabetically sorted?

Reply:
Correct.

Reply:
Done